dotnet/Cluster.cs

```diff
@@ -158,6 +158,12 @@ public InputMap<string> ClusterTags
         [Input("desiredCapacity")]
         public Input<int>? DesiredCapacity { get; set; }
 
+        /// <summary>
+        /// Disable creating the VPC CNI.
+        /// </summary>
+        [Input("disableVpcCni")]
+        public Input<bool>? DisableVpcCni { get; set; }
```

Comment:
I don't believe this flag actually disabled CNI. It just means that this package doesn't overwrite the default CNI with a version defined by this package. Worth validating the exact behavioral change when this is applied - but I suspect the flag should be something more like `useDefaultVpcCni?: bool`.

Reply:
That's correct -- you'll still get the default CNI that EKS provides. I guess the problem is this package has an attribute `vpcCniOptions` which suggests the ability to customize the CNI, which is true, but it's also not clear that you'll end up with a CNI manifest that's diverged from the AWS one. I used the term 'disable' here because I wanted to indicate that you're disabling what's in this package and getting what's the default. But I also see what you mean where "disable" suggests you're disabling the default as well. I worry about `useDefaultVpcCni` being awkward because we still have `vpcCniOptions` next to it, but maybe this can be explained via doc string.